linode_metadata/metadata_client.py

```diff
@@ -208,6 +246,8 @@ def __init__(
         timeout=DEFAULT_API_TIMEOUT,
         managed_token=True,
         managed_token_expiry_seconds=3600,
+        debug=False,
+        debug_file=None,
```

Comment:
I think the docstring for `MetadataClient` needs to be updated with these new fields 👍